python/plugins/MetaSearch/search_backend.py

```diff
@@ -84,6 +89,8 @@ def __init__(self, url, timeout, username, password, auth):
         self.record_info_template = 'record_metadata_dc.html'
         self.constraints = []
 
+        log_message(QCoreApplication.translate(
+            'MetaSearch', f'Connecting to CSW: {self.url}'), Qgis.Info)
```

Comment:
Fstrings doesn't work with translations I think. Same comment as @nyalldawson, the string will be collected in advance, but with fstring, the string will change. I personally keep using `.format`. Am I wrong ?

Side note, I thought we do not translate logs in QGIS, or in general. Logs is for advanced users or developers. Imagine you receive logs in another language, it makes it very difficult to debug.

Reply:
Thanks @Gustry.  Removed translation functionality from log messages.